CodeNow/runnable-angular
# SAN-2113 fix routing

No longer WIP! This is ready.

To test:
1. Navigate somewhere
2. Reload the page
3. Navigate everywhere else
4. Click all the buttons and make sure everything is displayed.
